org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
```

Comment:
I merged #18 since it has been sitting for a while and it had a cool :shipit: cat :). I suggest if any changes need to be made there we add the stub implementations in this PR, then Ravi can update the stubs with full implementations later in a later PR. Cool?